src/client/pages/FinancieleHulp/FinancieleHulp.tsx

```diff
@@ -0,0 +1,163 @@
+    const budgetbeheer = FINANCIELE_HULP.content?.deepLinks?.budgetbeheer;
+    if (!budgetbeheer) {
+      return [];
+    }
+
+    return {
+      ...budgetbeheer,
+      url: (
+        <LinkdInline external={true} href={budgetbeheer.url}>
+          Bekijk uw lening
+        </LinkdInline>
+      ),
+    };
+  }, [FINANCIELE_HULP.content]);
+  return (
+    <OverviewPage className={styles.FinancieleHulp}>
+      <PageHeading
+        backLink={{
+          to: AppRoutes.HOME,
+          title: 'Home',
+        }}
+        isLoading={isLoading(FINANCIELE_HULP)}
+        icon={<ChapterIcon />}
+      >
+        {ChapterTitles.FINANCIELE_HULP}
+      </PageHeading>
+      <PageContent>
+        <p>
+          Een online plek waar u alle informatie over uw geldzaken kunt vinden
+          als klant van Budgetbeheer (FIBU).
+        </p>
+        <p>
+          <Linkd
+            external={true}
+            href="https://www.amsterdam.nl/werk-inkomen/kredietbank-amsterdam/"
+          >
+            Meer informatie over de Kredietbank
+          </Linkd>
+        <br />
+          <Linkd
+            external={true}
+            href="https://www.amsterdam.nl/werk-inkomen/bijstandsuitkering/budgetbeheer/krefia-fibu/"
+          >
+            Meer informatie over FiBu
+          </Linkd>
+        </p>
+      </PageContent>
+      {schuldregelingen && (
+        <SectionCollapsible
+          id="SectionCollapsible-financiele-hulp-schuldregeling"
+          title="Schuldregeling"
+          hasItems={!!schuldregelingen}
+          startCollapsed={!!schuldregelingen}
+          className={styles.SectionBorderTop}
+          isLoading={isLoading(FINANCIELE_HULP)}
+          track={{
+            category: 'Financiele hulp overzicht / Schuldregeling',
+            name: 'Datatabel',
+          }}
+        >
+          <Table
+            className={styles.HulpTable}
+            displayProps={DISPLAY_PROPS}
+            items={[schuldregelingen]}
+          />
+        </SectionCollapsible>
+      )}
+      {!!leningen && (
+        <SectionCollapsible
+          id="SectionCollapsible-financiele-hulp-leningen"
+          title="Leningen"
+          hasItems={!!leningen}
+          startCollapsed={!schuldregelingen && !!leningen}
+          className={styles.SectionCollapsibleCurrent}
+          isLoading={isLoading(FINANCIELE_HULP)}
+          track={{
+            category: 'Financiele hulp overzicht / Leningen',
+            name: 'Datatabel',
+          }}
+        >
+          <Table
+            className={styles.HulpTable}
+            displayProps={DISPLAY_PROPS}
+            items={[leningen]}
+          />
+        </SectionCollapsible>
+      )}
+      {!!budgetbeheer && (
+        <SectionCollapsible
+          id="SectionCollapsible-financiele-hulp-budgetbeheer"
+          title="Financieel budgetbeheer"
+          hasItems={!!budgetbeheer}
+          startCollapsed={!schuldregelingen && !leningen && !!budgetbeheer}
```

Comment:
```suggestion
          startCollapsed={!!schuldregelingen || !!budgetbeheer}
```

Reply:
We willen toch alleen de bovenste default laten uitklappen?

Reply:
Ja klopt, het moet zijn `startCollapsed={!!schuldregelingen}`

Reply:
Dan toch:
`startCollapsed={!schuldregelingen || !leningen}`

Reply:
Nee, `collapsed` betekend `ingeklapt` dus:

start ingeklapt als er schuldregelenen of budgetbeheer is